docs/requirements/apod/100-esx-layer1-lag0.robot

```diff
@@ -0,0 +1,18 @@
+*** Settings ***
+Documentation     aPod node LAG0 connectivity.
```

Comment:
Might also be usefull to mention which of the physical ports are used to form a LAG.

Reply:
as discussed physical ports to LAG relation will be modelled in Netbox.